### ISSUE TITLE
GPU memory 

### ISSUE THREAD

**Issue by emrekeles-arch** [batmanlab/Mammo-CLIP#3]
How much GPU memory is used during training and how long does training take?

**Comment by shantanu-ai** [comment]
We train it using 1 NVIDIA RTX6000 GPU for 10 epochs. It took 3 days for UPMC data (image-text). It took around 4.5 days for UPMC (image-text) + VinDr (image-labels) data. We are now migrating to distributed parallel to pre-train with bigger datasets.
